src/main/java/org/spongepowered/common/command/parameter/managed/standard/SpongeWorldPropertiesValueParameter.java

```diff
@@ -60,7 +60,7 @@ public ResourceKey getKey() {
     @Override
     @NonNull
     public List<String> complete(@NonNull final CommandContext context) {
-        return SpongeCommon.getGame().getServer().getWorldManager().getAllProperties().stream().map(WorldProperties::getKey)
+        return SpongeCommon.getGame().getServer().getWorldManager().getAllProperties().stream().filter(properties -> properties.getWorld().isPresent()).map(WorldProperties::getKey)
```

Comment:
@dualspiral I feel we should discuss this because one can make an argument for this parameter giving you all the properties as much as someone arguing it should be only ones that are "online".

Maybe we need two builtin API parameters? ONLINE and ALL?

Reply:
I'm glad you asked. 

If we are to have one element, it should get all worlds, unloaded or otherwise. Having two elements does make sense in that regard because there are (obviously) reasons why you might just want to target online worlds.

This then pertains to a wider question - should the location argument only allow for online worlds to be targetted? I'd be inclined to say yes, but it might be that we want to have both here too.